static/templates/recent_topics_filters.hbs

```diff
@@ -1,4 +1,12 @@
 <button data-filter="all" type="button" class="btn btn-default btn-recent-filters">{{t 'All' }}</button>
+<button data-filter="include_private" type="button" class="btn btn-default btn-recent-filters {{#if is_spectator}}fake_disabled_button{{/if}}" role="checkbox" aria-checked="true">
+    {{#if (or filter_pm not is_spectator)}}
```

Comment:
This should be:
```
-    {{#if (or filter_pm not is_spectator)}}
+    {{#if filter_pm}}
```

`filter_pm` is always false if a spectator, so it doesn't accomplish anything, and this tweak better matches the other filters which all have the same property.